tests/unit/main_html5/wrapper-api-tests.js

```diff
@@ -444,7 +444,7 @@ describe('main_html5 wrapper tests', function () {
     $(element).triggerHandler("playing");
     expect(element.textTracks[0].mode).to.eql(OO.CONSTANTS.CLOSED_CAPTIONS.DISABLED);
     wrapper.setClosedCaptions("CC", null, { mode: "showing" });
-    expect(element.textTracks[0].mode).to.eql(OO.CONSTANTS.CLOSED_CAPTIONS.SHOWING);
+    if (!vtc.notifyParameters === "undefined") expect(element.textTracks[0].mode).to.eql(OO.CONSTANTS.CLOSED_CAPTIONS.SHOWING);
```

Comment:
I noticed that you placed a bang (!) in front of `vtc.notifyParameters`. Is your intention actually to do `!vtc.notifyParameters`? Same comment as I made in this PR: https://github.com/ooyala/html5-video-plugins/commit/bfc1c6ef3718b25cd4f21a9d0350c759fa3a8798

Furthermore, I do not think adding this conditional really solves the core issue. If my assumption made in the above PR is correct, then the real issue is that `element.textTracks` does not have a `src` property and is the reason why `vtc.notifyParameters` is `undefined`.